bin/pycbc_multi_inspiral

```diff
@@ -901,15 +928,20 @@ with ctx:
                         # Calculate chisq reweighted SNR
                         if nifo > 2:
                             reweighted_snr = ranking.newsnr(
-                                rho_coh, network_chisq_dict
+                                rho_coh, network_chisq_dict,
+                                q=args.chisq_index, n=args.chisq_nhigh
```

Comment:
I suggest doing another pass through `black` to properly format these edited calls.

Reply:
The latest commit applied another round of `black`.